source/renderer/app/features/menu-updater/MenuUpdater.js

```diff
@@ -0,0 +1,35 @@
+// @flow
```

Comment:
I think this file should be renamed to `context.js` as it will be the feature provider if not mistaken.

Reply:
I didn't call it `context` since it does not provide anything through context so IMO it should not be called this way.